package.json

```diff
@@ -45,6 +45,8 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "devDependencies": {
-    "prettier": "^1.15.3"
+    "autoprefixer": "^10.4.0",
+    "gatsby-plugin-postcss": "^5.2.0",
```

Comment:
What are you using postcss plugin for? I thought you were just using tailwind.

Reply:
Now I realize that I'm using gatsby-plugin-postcss to give the style i want to. Both work together (TailwindCSS and gatsby-plugin-postcss).  In the postcss.config.js file you can see that i'm using tailwindcss as a postcss plugin.

Reply:
I'm not using "prettier" anymore. I need to use "autoprefixer" to parse css and add css vendor prefixes, this is necessary to use the postcss plugin.